packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/node/FlowNode.vue

```diff
@@ -390,6 +392,12 @@ $io-column-min-width-no-fields: 70px;
   min-width: $node-min-width;
   padding: $kui-space-40 0;
 
+  &.readonly {
+    .menu {
+      visibility: hidden;
+    }
+  }
```

Comment:
Should we find another way to disable the menu on nodes? It seems I'm still able to trigger the menu by clicking somewhere in the top-right corner of a node:

<img width="414" height="191" alt="Image" src="https://github.com/user-attachments/assets/652439e6-5f41-408e-b76f-76e46bca02d7" />